lib/build.gradle

```diff
@@ -31,7 +31,17 @@ android {
 
 dependencies {
 
+    implementation 'androidx.annotation:annotation:1.4.0'
     testImplementation 'junit:junit:4.13.2'
     androidTestImplementation 'androidx.test.ext:junit:1.1.3'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.4.0'
+
+    implementation(
+            "com.google.android.gms:play-services-mlkit-text-recognition:17.0.0",
+            "org.jmrtd:jmrtd:0.7.9",
+            "com.madgag.spongycastle:prov:1.58.0.0",
```

Comment:
@AVI5HEK  I didn't see where we will use these `dependencies` :
`com.madgag.spongycastle:prov` and `net.sf.scuba:scuba-sc-android` May I know why we need them 🙏

Reply:
@Wadeewee you observed something which led to an interesting discovery. We didn't use a class in `spongycastle` when converting from Java code from the POC project to Kotlin. 🤔 For `scuba`, it's right [here](https://github.com/nimblehq/alpine/pull/8/files#diff-d55eddddfc88766326c15b54e97d83d6149b3b466e8720307284ef75447c4e72R12).